src/common/decorators/validation.ts

```diff
@@ -42,6 +42,33 @@ export function IsTicker(validationOptions?: ValidationOptions) {
   );
 }
 
+const assetIdRegex = /0x[0-9A-F]{32}/;
```

Comment:
```suggestion
const assetIdRegex = /0x[0-9a-fA-F]{32}/;
```

I missed lower case support in my initial comment 😅